src/components/ConfigEditor.tsx

```diff
@@ -74,31 +68,22 @@ export function ConfigEditor(props: Props) {
       </InlineField>
       <br />
       <InlineField label="Org ID" labelWidth={17}>
-        <Input
-            value={jsonData.orgID || ''}
-            placeholder=""
-            width={40}
-            onChange={onOrgIDChange}
-        />
+        <Input value={jsonData.orgID || ''} placeholder="" width={40} onChange={onOrgIDChange} />
       </InlineField>
       <br />
-      { window.location.hostname === 'localhost' || window.location.hostname.endsWith('axiomtestlabs.co') ? (
-        <div>
-          <Label description="The Axiom host to use. Leave the default value if you are not using a self-hosted Axiom instance.">
-            <h6>Axiom Host</h6>
-          </Label>
-          <InlineField label="URL" labelWidth={17}>
-            <Input
-              onChange={onHostChange}
-              value={jsonData.apiHost || 'https://api.axiom.co'}
-              placeholder="axiom host"
-              width={40}
-            />
-          </InlineField>
-        </div>
-      ) : (
-        ''
-      )}
+      <div>
+        <Label description="The Axiom host to use.">
+          <h6>Axiom Host</h6>
+        </Label>
+        <InlineField label="URL" labelWidth={17}>
+          <Input
+            onChange={onHostChange}
+            value={jsonData.apiHost || 'https://api.axiom.co'}
```

Comment:
can we make this a drop down with api.axiom.co and api.eu.axiom.co later?

Reply:
hmmm sometimes we need to test against dev or localhost, but yeah I think we can have different fields maybe, show users one with a dropdown 👍